### DISCUSSION TITLE
Add file to entry

### DISCUSSION THREAD

**Discussion by MarkStephan89** [dani-garcia/vaultwarden#4492] [Q&A] [answered]
hi, 
it is possible to add an file to an entry?
For example the German Elster cert?

Thanks!

**Comment by Masgalor** [comment, marked as answer]
See https://bitwarden.com/help/attachments/ or in German  https://bitwarden.com/de-de/help/attachments/ .

**Reply by MarkStephan89** [reply]
that was simple! thanks!
